Update reference file docs section (#666)
sketch-hq/SketchAPI: docs/pages/file-format.md

zip -r -X document.sketch *
```

## JSON Schema

The [JSON Schema](https://github.com/sketch-hq/sketch-file-format) based specification describes how Sketch documents are stored on disk, and how this changes from version to version.

> We hope this helps everyone building low level integrations of Sketch, such as server and command line applications that create and maniupulate Sketch files outside the Mac app.

Reviewer: I'd adjust the tone of voice, less narrator, more instructive, descriptive:

```suggestion
Build low level integrations of Sketch such as server and command-line applications. Read and generate…
```

## Typescript types

The TypeScript [types](https://github.com/sketch-hq/sketch-file-format-ts) model Sketch file JSON, and are automatically generated from the file format [json schema](/file-format/reference/json-schema) specification.

> The types will allow you to strongly type representations and fragments of Sketch documents in TypeScript projects.

Reviewer: Similar, more direct wording making it clear that it has types for all fragments such as artboards, layers, etc.
